src/prisma/schema.prisma

```diff
@@ -965,6 +965,7 @@ model Application {
   walletAddress      String?
   rewardInfo         String?
   message            String?
+  messageNodes       String?
```

Comment:
Approved at first, but this should be a JSON column with null as the default. I haven't looked at the PR that uses it but we should work around whatver issues were making this be a string